03_letters/CameronSchafer/letters.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+// string to rune with counts
+func letters(s string) map[rune]int {
+	tMap := make(map[rune]int)
+
+	for i := range s {
+		tMap[rune(s[i])]++
+	}
+
+	return tMap
+}
+
+func sortLetters(m map[rune]int) []string {
+	tSlice := []int{}
+	for i := range m {
+		tSlice = append(tSlice, int(i))
+	}
+
+	// sort the ints
+	sort.Ints(tSlice)
+
+	sSlice := []string{}
+	for _, val := range tSlice {
+		sSlice = append(sSlice, fmt.Sprintf("%c:%d", val, m[rune(val)]))
```

Comment:
Nice use of "%c", I couldn't find how to print out my rune correctly like this, and my solution was a bit less elegant ;)